AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Configuration/DataStoreConfiguration+Helper.swift

```diff
@@ -15,6 +15,41 @@ extension DataStoreConfiguration {
     public static let defaultSyncMaxRecords: UInt = 10_000
     public static let defaultSyncPageSize: UInt = 1_000
 
+    #if os(watchOS)
+    /// Creates a custom configuration. The only required property is `conflictHandler`.
+    ///
+    /// - Parameters:
+    ///   - errorHandler: a callback function called on unhandled errors
+    ///   - conflictHandler: a callback called when a conflict could not be resolved by the service
+    ///   - syncInterval: how often the sync engine will run (in seconds)
+    ///   - syncMaxRecords: the number of records to sync per execution
+    ///   - syncPageSize: the page size of each sync execution
+    ///   - authModeStrategy: authorization strategy (.default | multiauth)
```

Comment:
Missing doc for the `disableSubscriptions` parameter

Reply:
updating